app/main.py

```diff
@@ -1,6 +1,35 @@
+import os
+import requests
+
+
+URL = "http://api.weatherapi.com/v1/current.json"
+FILTERING = "Paris"
+API_KEY = os.environ.get("WEATHER_API_KEY")
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    response = requests.get(
+        URL,
+        params={
+            "key": API_KEY,
+            "q": FILTERING,
+        }
+    )
+    if response.status_code == 200:
+        data = response.json()
+        print(f"Performing request to Weather API for city "
```

Comment:
```suggestion
        print("Performing request to Weather API for city "
```
don't overuse f-strings